spec/features/crop_searches_spec.rb

```diff
@@ -29,7 +29,7 @@
     FactoryGirl.create_list(:crop, 10)
     Crop.searchkick_index.refresh
     click_button 'Search'
-    expect(page).to have_content("no crops matching")
+    expect(page).to have_content("any crops matching")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.